### DISCUSSION TITLE
【第二次课】问答交流与开源专家受聘仪式

### DISCUSSION THREAD

**Discussion by will-ww** [X-lab2017/OSSDevGov2021#7] [任意分享]
![受聘2](https://user-images.githubusercontent.com/15010826/110324035-6844a400-8050-11eb-9a25-8eab2c5ac78c.jpg)
![01](https://user-images.githubusercontent.com/15010826/110323995-5c58e200-8050-11eb-821c-0f837a25746d.jpg)
![02](https://user-images.githubusercontent.com/15010826/110324009-6084ff80-8050-11eb-9dcb-52399f5759f8.jpg)
![03](https://user-images.githubusercontent.com/15010826/110324013-637ff000-8050-11eb-8411-ce965c629d2f.jpg)
![04](https://user-images.githubusercontent.com/15010826/110324022-65e24a00-8050-11eb-93a5-6a282ad16f4f.jpg)

**Comment by xiaoya-yaya** [comment]
### 第二次课课程资源
#### 课件
[02《开源软件开发与社区治理》第2讲.pdf](https://github.com/X-lab2017/OSSDevGov2021/files/6105821/02.2.pdf)

#### 实训Lab
[开源实训Lab-01：软件开发基础.pdf](https://github.com/X-lab2017/OSSDevGov2021/files/6105827/Lab-01.pdf)
